mabl snippets/ValueFromCommaSeparatedList.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * TITLE: Value from comma separated list
+ * NOTE: This is a 1-indexed list. If you want the first value from the list, set "index" to "1"
```

Comment:
curious why it's index 1?

Reply:
the variable run.loop_index starts at 1 so this will help with looping customers.